README.md

```diff
@@ -6,25 +6,26 @@ Benchmarking DataFusion and DuckDB over [ClickBench](https://benchmark.clickhous
 * DataFusion 31.0.0
 * DuckDB 0.8.1
 
+## Results
+All results are checked in to [results]
+
+The scripts in this repository run queries via python bindings for both DataFusion and DuckDB
+
 ## Setting up the Environment
 
 ```bash
-# install datafusion
-git clone https://github.com/apache/arrow-datafusion.git
-cd arrow-datafusion
-git checkout 31.0.0
-cargo install --profile release --path datafusion-cli
-
-# install duckdb
-git clone https://github.com/duckdb/duckdb
-cd duckdb
-git checkout v0.8.1
-BUILD_BENCHMARK=1 BUILD_TPCH=1 make -j$(nproc)
 
-# Setup Python virtual environment
+# Setup Python virtual environment and databases
 python3 -m venv venv
 source venv/bin/activate
 pip install pyarrow pandas matplotlib seaborn
+
+# install duckdb
```

Comment:
I think having the systems installed as part of setup rather than each time the queries are run makes it much less confusing (and easier to control and understand what is being tested)